### PR TITLE
Make more examples exportable to codepen

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Make Persona examples exportable to codepen, and get rid of all other references to _exampleStyles.scss (making the examples touched exportable when possible).

Also start converting some examples to function components.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9468)

### CODE REVIEW

**Comment by ecraig12345** [comment]
> I was wondering if you could remove any exclusions in `lint-imports.js` but if these changes are just for CSS imports, I don't think you can. (CSS imports aren't disallowed by `lint-imports`.)

I thought about trying to get rid of some of the TestImages references (that's the lowest-hanging fruit), but that would require repeating the CDN URL everywhere.

A slight improvement would be repeating the CDN URL everywhere and making it type-safe as follows, but this is kind of gross too...
```
// in some shared file
export type FabricAssetsCdnUrl = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/';

// in examples
import { FabricAssetsCdnUrl } from '../wherever';

const cdnUrl: FabricAssetsCdnUrl = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/';
```

**Comment by ecraig12345** [comment]
Or this:
```ts
// in some shared file
export type TestImagePersonaFemaleUrl = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-female.png';

// in examples
import { TestImagePersonaFemaleUrl } from '../wherever';

const imageUrl: TestImagePersonaFemaleUrl = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-female.png';
```

File: apps/fabric-website/src/pages/Controls/ButtonPage/ButtonPage.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
-import { Checkbox } from 'office-ui-fabric-react';
+import { Checkbox, ICheckboxStyles } from 'office-ui-fabric-react/lib/Checkbox';
```
Comment (Vitalius1) [inline apps/fabric-website/src/pages/Controls/ButtonPage/ButtonPage.tsx]: Having 2 exact same pages under different packages just creates risk for inconsistency. Just curious of why not reusing the `fabric-website` for our local development? Why have two sets of code to constantly update?

Comment (ecraig12345) [reply apps/fabric-website/src/pages/Controls/ButtonPage/ButtonPage.tsx]: At this point it's historical reasons that we haven't had time to fix. :( Totally agree with your point though.

**Pull request merged**